### PR TITLE
fix(topic-teaser): content-indentation, image aspect ratios, font sizes and paddings

### PR DESCRIPTION
**Pull request by oliverschuerch**
In the previous PR #162 some changes were made that needed to be fixed.
- One-sided negative content indentation on larger screens.
Since we had to extend the col-classes anyway - resulting in a breakingchange - we decided to move the col-definition of the containers .topic-teaser-content and .topioc-teaser-image-container from html to css.
- Incorrect image aspect ratios
- Font sizes and padding for link-list links on smaller screens

### CODE REVIEW

File: .changeset/mean-cats-melt.md [hunk 1]
```diff
@@ -2,5 +2,5 @@
 '@swisspost/web-styles': major
 ---
 
-Removed option-variables such as $isolate-components and $variable-prefix.
-Moved bootstrap-feature variables ($enable-*). These will no longer be forwarded.
\ No newline at end of file
+Removed option-variables such as $isolate-components.
+Moved bootstrap-feature variables ($enable-\*). These will no longer be forwarded.
```
Comment (gfellerph) [inline .changeset/mean-cats-melt.md:6]:
```suggestion
Moved bootstrap-feature variables ($enable-\*) to a separate file. These will no longer be forwarded and cannot be overriden.
```

File: packages/web-demo/src/app/post-sample/components/topic-teaser/topic-teaser-demo/topic-teaser-demo.component.html [hunk 2]
```diff
@@ -3,16 +3,16 @@
...
         </div>
 
-        <div class="topic-teaser-content col-12 col-lg-8">
+        <div class="topic-teaser-content">
           <h2 class="topic-teaser-title font-curve-large mb-large">
             <span class="bold">Lorem ipsum</span>
             <span class="light">sit amet, consetetur sadipscing elitr</span>
           </h2>
-          <ul class="link-list font-curve-regular mb-large">
+          <ul class="link-list font-curve-small mb-large">
```
Comment (gfellerph) [inline packages/web-demo/src/app/post-sample/components/topic-teaser/topic-teaser-demo/topic-teaser-demo.component.html:15]: The link list font-size should be 20px on desktop:

https://www.figma.com/file/xZ0IW0MJO0vnFicmrHiKaY/Components?node-id=174%3A200

File: packages/web-demo/src/app/post-sample/components/topic-teaser/topic-teaser-right-demo/topic-teaser-right-demo.component.html [hunk 3]
```diff
@@ -3,12 +3,12 @@
...
       <div class="row pt-huge-r">
 
-        <div class="topic-teaser-content col-12 col-lg-8">
+        <div class="topic-teaser-content">
           <h2 class="topic-teaser-title font-curve-large mb-large">
             <span class="bold">Lorem ipsum</span>
             <span class="light">sit amet, consetetur sadipscing elitr</span>
           </h2>
-          <ul class="link-list font-curve-regular mb-large">
+          <ul class="link-list font-curve-small mb-large">
```
Comment (gfellerph) [inline packages/web-demo/src/app/post-sample/components/topic-teaser/topic-teaser-right-demo/topic-teaser-right-demo.component.html:11]: The link list font-size should be 20px on desktop:

https://www.figma.com/file/xZ0IW0MJO0vnFicmrHiKaY/Components?node-id=174%3A200

File: packages/web-demo/src/app/post-sample/components/topic-teaser/topic-teaser-right-demo/topic-teaser-right-demo.component.html [hunk 4]
```diff
@@ -17,8 +17,8 @@ <h2 class="topic-teaser-title font-curve-large mb-large">
           </ul>
         </div>
 
-        <div class="topic-teaser-image-container col-10 col-rg-8 col-lg-4">
-          <img class="topic-teaser-image" src="assets/images/gazelle.jpg" alt="Gazelle" />
```
Comment (gfellerph) [inline packages/web-demo/src/app/post-sample/components/topic-teaser/topic-teaser-right-demo/topic-teaser-right-demo.component.html:21]: If these images are no longer used, we should delete them from the assets and the [attribution file](https://github.com/swisspost/common-web-frontend/blob/main/packages/web-demo/src/assets/images/ATTRIBUTION.md).

**Review by gfellerph — CHANGES_REQUESTED** [review]

File: packages/web-demo/src/app/post-sample/components/topic-teaser/topic-teaser-demo/topic-teaser-demo.component.html (hunk 2 above)
Comment (oliverschuerch) [reply packages/web-demo/src/app/post-sample/components/topic-teaser/topic-teaser-demo/topic-teaser-demo.component.html:15]: True, true...
But there is no matching font-curve for the new topic-teaser design on figma.

| Breakpoint | font-size@Figma | font-size@font-curve-regular | font-size@font-curve-small |
| :-: | -: | -: | -: |
| xs | 14px | 17px | 15px |
| sm | 14px | 17px | 15px |
| rg | 14px | 17px | 16px |
| md | 14px | 17px | 16px |
| lg | 20px | 20px | 17px |
| xl | 20px | 20px | 17px |
| xxl | 20px | 20px | 17px |

File: packages/web-demo/src/app/post-sample/components/topic-teaser/topic-teaser-right-demo/topic-teaser-right-demo.component.html (hunk 4 above)
Comment (oliverschuerch) [reply packages/web-demo/src/app/post-sample/components/topic-teaser/topic-teaser-right-demo/topic-teaser-right-demo.component.html:21]: done

**Review by gfellerph — APPROVED** [review]

File: packages/web-demo/src/app/post-sample/components/topic-teaser/topic-teaser-demo/topic-teaser-demo.component.html (hunk 2 above)
Comment (oliverschuerch) [reply packages/web-demo/src/app/post-sample/components/topic-teaser/topic-teaser-demo/topic-teaser-demo.component.html:15]: Do it custom

**Pull request merged**